cmd/fyne_demo/main.go

```diff
@@ -159,6 +160,13 @@ func makeMenu(a fyne.App, w fyne.Window) *fyne.MainMenu {
 	)
 }
 
+func supportedTutorial(t tutorials.Tutorial) bool {
+	if !t.SupportWeb && fyne.CurrentDevice().IsBrowser() {
+		return false
+	}
+	return true
```

Comment:
Hmm, it's weird that staticcheck didn't catch this. You should be able to remove the if-statement and just return the negated boolean value of the result you get from `!t.SupportWeb && fyne.CurrentDevice().IsBrowser()`.

It could also be better to call this `unsupportedTutorial` given that it is used with !supported everywhere.